frontend/package.json

```diff
@@ -35,6 +35,7 @@
     "expo-status-bar": "~1.11.1",
     "jest": "^29.2.1",
     "jest-expo": "^50.0.2",
+    "moment": "^2.30.1",
```

Comment:
What is `moment` do in our project?

Reply:
https://github.com/hyunghoseo/nykea-app/pull/64/files#diff-cdeecc7d68e0fb403af97341cb0f958252ab04ef6f6090d993592918e1fc64beR59

moment module allows us to have the date or time in a specific format. 

You can read more about it here : https://momentjs.com